### PR TITLE
[Mobile presentation]question detail done

### PR DESCRIPTION
**Pull request by abenezer444**

### CODE REVIEW

File: Mobile/lib/presentation/pages/questions.detail/questions_detail.dart [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+                    height: 50,
+                    child: TextField(
+                      decoration: InputDecoration(
+                        suffixIcon: IconButton(
+                            onPressed: () {}, icon: Icon(Icons.send)),
+                        border: InputBorder.none,
+                        hintText: "reply",
+                      ),
+                    ),
+                  ),
```
Comment (amanyih) [inline Mobile/lib/presentation/pages/questions.detail/questions_detail.dart:226]: Screenshot (112)

when the profile is touched it should have a little menu and let the user choose if he wants to answer as anonymous or not

File: Mobile/lib/presentation/pages/questions.detail/questions_detail.dart [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        .toList();
+    return Scaffold(
+      appBar: AppBar(title: Text('question comments')),
+      bottomNavigationBar: BottomAppBar(),
+      body: Column(
+        children: [
+          SizedBox(
+            height: 10,
+          ),
+          QuestionW(question: question),
```
Comment (amanyih) [inline Mobile/lib/presentation/pages/questions.detail/questions_detail.dart:191]: When giving heights use `height:10.h` 
first import` import 'package:flutter_screenutil/flutter_screenutil.dart';`

File: Mobile/lib/presentation/pages/questions.detail/questions_detail.dart (hunk 1 above)
Comment (amanyih) [reply Mobile/lib/presentation/pages/questions.detail/questions_detail.dart:226]: and also give space between the question and answer.

**Review by amanyih — CHANGES_REQUESTED** [review]
check this comments

File: Mobile/lib/presentation/pages/questions.detail/questions_detail.dart (hunk 1 above)
Comment (abenezer444) [reply Mobile/lib/presentation/pages/questions.detail/questions_detail.dart]: check it out esti

**Review by Merwan-J — APPROVED** [review]

**Review by Merwan-J — APPROVED** [review]

**Pull request closed without merging**